<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Organizer</class>
 <widget class="QMainWindow" name="Organizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Organizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="fileTypesTree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>211</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>File Types</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File Extensions</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Books</string>
     </property>
     <property name="text">
      <string>*.pdf, *.chm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Videos</string>
     </property>
     <property name="text">
      <string>*.ogg, *.ogv, *.mpg, *.mp4, *.mov, *.avi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Documents</string>
     </property>
     <property name="text">
      <string>*.doc, *.odt, *.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Archives</string>
     </property>
     <property name="text">
      <string>*.rar, *.zip, *.tar, *.tar.gz</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="pathEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="keywordsEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="keywordsTree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>211</width>
      <height>471</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditKeyPressed</set>
    </property>
    <column>
     <property name="text">
      <string>Keywords</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="fileListTree">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>961</width>
      <height>881</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>File Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File Path</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select A Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>keywordsEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>Organizer</receiver>
   <slot>addNewKeyword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileTypesTree</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Organizer</receiver>
   <slot>startSearch(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileListTree</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Organizer</receiver>
   <slot>openFileOrDirectory(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keywordsTree</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>Organizer</receiver>
   <slot>showKeywordsTreeMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>Organizer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileListTree</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>Organizer</receiver>
   <slot>showFilesTreeMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Organizer</receiver>
   <slot>showFileDialog(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keywordsTree</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Organizer</receiver>
   <slot>editKeyword(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>722</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>721</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNewKeyword()</slot>
  <slot>startSearch(QTreeWidgetItem*,int)</slot>
  <slot>openFileOrDirectory(QTreeWidgetItem*,int)</slot>
  <slot>showKeywordsTreeMenu(QPoint)</slot>
  <slot>showFilesTreeMenu(QPoint)</slot>
  <slot>showFileDialog(bool)</slot>
  <slot>editKeyword(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
